### PR TITLE
Fix #114: Provide valid options for CSSComb config

### LINKED ISSUE #114: Invalid options for CSSComb CLI
Steps:
1. Grab the latest `.csscomb.json` config from the code-style repo
2. Ensure you have the `csscomb` npm module & bin `npm install -g csscomb`
3. Run CSSComb against some SCSS files using the `.csscomb.json` config:
`csscomb -c node_modules/mobify-code-style/css/.csscomb.json src/sass/`

Result:
Errors for invalid options appear:
csscomberrors

Expected:
CSSComb runs without any invalid option errors.

### PR DESCRIPTION
**Pull request by marlowpayne**
## Changes
- Fixes #114 by removing invalid options from CSSComb config

## How To Test
- Checkout this change
- Reference it when running CSSComb on the command line:
`csscomb -c node_modules/mobify-code-style/css/.csscomb.json src/sass/`
- Make sure that no invalid option errors surface
- Try it in a text editor (Internal [wiki link](https://mobify.atlassian.net/wiki/display/LT/CSSComb) for setting up CSSComb in your editor)

## TODOs:
- [x] +1
- [x] ~~Updated README~~
- [x] Updated CHANGELOG

### CODE REVIEW

File: css/.csscomb.json [hunk 1]
```diff
@@ -13,11 +13,9 @@
     "combinator-space": [" ", " "],
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": null,
```
Comment (jeffkamo) [inline css/.csscomb.json]: Should this be removed, or just set to `false`?

File: css/.csscomb.json [hunk 2]
```diff
@@ -13,11 +13,9 @@
...
     "rule-indent": "    ",
     "stick-brace": " ",
     "unitless-zero": true,
-    "quotes": false,
```
Comment (jeffkamo) [inline css/.csscomb.json]: Should this be removed, or just set to our preferred setting? Which [should be `single`](https://github.com/mobify/mobify-code-style/tree/develop/css/css-best-practices#format)

File: css/.csscomb.json (hunk 1 above)
Comment (marlowpayne) [inline css/.csscomb.json]: There was a previous issue with that. Ref #81

File: css/.csscomb.json (hunk 2 above)
Comment (marlowpayne) [inline css/.csscomb.json]: We wanted to support both, but couldn't. Ref #79

File: css/.csscomb.json (hunk 1 above)
Comment (jeffkamo) [inline css/.csscomb.json]: Fair point. To be thorough, is this rule also disabled in our sass-lint ruleset?

File: css/.csscomb.json (hunk 2 above)
Comment (jeffkamo) [inline css/.csscomb.json]: Roger that then 👍

File: css/.csscomb.json (hunk 1 above)
Comment (jeffkamo) [inline css/.csscomb.json]: Perhaps we should disable [this rule](https://github.com/mobify/mobify-code-style/blob/develop/css/.sass-lint.yml#L56)

Comment (marlowpayne) [inline css/.csscomb.json]: Yup, they are both disabled. From the [rules doc](https://github.com/sasstools/sass-lint#rules):

> For all rules, setting their severity to 0 turns it off

**Comment by jeffkamo** [comment]
👍

**Pull request merged**